project/Dependencies.scala

```diff
@@ -340,7 +340,7 @@ object Dependencies {
 
   val Sns = Seq(
     libraryDependencies ++= Seq(
-      "com.github.matsluni" %% "aws-spi-akka-http" % "0.0.5" excludeAll ExclusionRule(
+      "com.github.matsluni" %% "aws-spi-akka-http" % "0.0.6" excludeAll ExclusionRule(
```

Comment:
Since this is already referenced twice (and maybe more in the future for other AWS modules), can you create a `AwsSpiAkkaHttpVersion` variable ?

Reply:
Done. See 85f60ad.